pkg/windows/windows.go

```diff
@@ -475,10 +476,17 @@ func (vm *windows) Bootstrap(desiredVer, watchNamespace, wicdKubeconfigContents
 	if err := vm.createDirectories(); err != nil {
 		return fmt.Errorf("error creating directories on Windows VM: %w", err)
 	}
+	if tlsPresent {
+		if err := vm.EnsureFileContent(tlsData["tls.crt"], "tls.crt", TlsDir); err != nil {
+			return fmt.Errorf("unable to transfer TLS certs: %w", err)
+		}
+		if err := vm.EnsureFileContent(tlsData["tls.key"], "tls.key", TlsDir); err != nil {
+			return fmt.Errorf("unable to transfer TLS key: %w", err)
+		}
```

Comment:
This should be done within transferFiles(). The files to transfer are setup as part of windows.New()